QUICKSTART.md

```diff
@@ -0,0 +1,308 @@
+# OpenTelemetry QuickStart
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Configuration](#configuration)
+    + [Sampler](#sampler)
+    + [Span Processor](#span-processor)
+    + [Exporter](#exporter)
+- [Tracing](#tracing)
+  * [Create basic Span](#create-basic-span)
+  * [Create nested Spans](#create-nested-spans)
+  * [Span Attributes](#span-attributes)
+  * [Create Spans with events](#create-spans-with-events)
+  * [Create Spans with links](#create-spans-with-links)
+  * [Context Propagation](#context-propagation)
+- [Metrics](#metrics)
+
+<!-- tocstop -->
+
+OpenTelemetry can be used to instrument code for collecting distributed traces and recording metrics.
+For more details, check out the [Specification Overview].
+
+In the following examples, we demonstrate how to configure the OpenTelemetry SDK, create spans and record metrics through the OpenTelemetry API.
+
+[Specification Overview]: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/overview.md
+
+# Configuration
+
+**Libraries** that want to export distributed tracing using OpenTelemetry only need a dependency on the `opentelemetry-api` package
+and should never configure OpenTelemetry themselves. The configuration must be provided by **Applications** which should also depend on the 
+`opentelemetry-sdk` package, or any other implementation of the OpenTelemetry API. This way, libraries will obtain a real tracer
+implementation only if the user application is configured for it. For more details, check out the [Library Guidelines].
+
+[Library Guidelines]: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/library-guidelines.md
+
+The application has to install a span processor with an exporter and may customize the behavior of the OpenTelemetry SDK.
+
+For example, a basic configuration instantiates the SDK tracer registry and sets to export the traces to a logging stream.
+
+```java
+// Get the tracer
+TracerSdkRegistry tracerRegistry = OpenTelemetrySdk.getTracerRegistry();
+
+// Set to export the traces to a logging stream
+tracerRegistry.addSpanProcessor(SimpleSpansProcessor.newBuilder(new LoggingExporter()).build());
+```
+
+### Sampler
+
+It is not always feasible to trace and export every user request in an application.
+In order to strike a balance between observability and expenses, traces can be sampled. 
```

Comment:
It might be helpful to link to https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/sdk-tracing.md#sampling here, but even that doc doesn't make it clear what happens to non-sampled traces (don't get exported, don't accept events, etc.).

Reply:
I very personally feel actual documentation (once we have an actual OTel website) could be better than the actual Specification, as it might be too dense for end users, as commented before.